web/src/components/LandingPageSlider.tsx

```diff
@@ -0,0 +1,176 @@
+import { LandingPageActionTile } from "@/components/LandingPageActionTile";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { QUERIES, ROUTES, routeWithQuery } from "@/lib/domain-logic/routes";
+import analytics from "@/lib/utils/analytics";
+import { useRouter } from "next/router";
+import { ReactElement, useRef, useState } from "react";
+import Slider from "react-slick";
+import "slick-carousel/slick/slick-theme.css";
+import "slick-carousel/slick/slick.css";
+import { useMountEffect } from "../lib/utils/helpers";
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+function SampleNextArrow(props: any) {
+  const { onClick } = props;
+  return (
+    <button onClick={onClick} className="slider-button next">
+      {">"}
+    </button>
+  );
+}
+function SamplePrevArrow(props: any) {
+  const { onClick } = props;
+  return (
+    <button onClick={onClick} className="slider-button prev">
+      {"<"}
+    </button>
+  );
+}
+
+export const LandingPageSlider = (): ReactElement => {
+  const router = useRouter();
+  const { Config } = useConfig();
+  const [activeSlide, setActiveSlider] = useState(0);
+  const [slideBreakPoint, setSlideBreakPoint] = useState(7);
+  const cardRef = useRef<null | HTMLButtonElement>(null);
+  const sliderRef = useRef<null | HTMLDivElement>(null);
+
+  const getNumberOfTilesToShow = (): number => {
+    const sliderWidth = sliderRef.current?.clientWidth;
+    const cardWidth = cardRef.current?.parentElement?.clientWidth;
+    if (!sliderWidth || !cardWidth) return 7;
+    const numberOfTilesToFit = Math.ceil(sliderWidth / (cardWidth + 100));
```

Comment:
100 is another magic number - what does it represent? why are we adding it?

Reply:
I'm confused by the naming here: 

this function is called "getNumberOfTilesToShow" and this variable is called "numberOfTilesToFit". Unless "fit" does not mean the same thing as "show" (which it seems to?) then I would expect "numberOfTilesToFit" to be the value returned from this function, given these names. But it's not - you return seven minus the "numberOfTilesToFit". Does that mean that this isn't actually the number of tiles to fit? If so, what *is* it?